### ISSUE TITLE
Tensorboard projector module not displaying

### ISSUE THREAD

**Issue by tbelhalfaoui** [guildai/guildai#106]
I am unable to use Tensorboard Projector module with guild.

Here is what I do:
* Run an operation that (only) pushes embeddings to Tensorboard
* Run `guild tensorboard`
* In Tensorboard, this specific tensorboard run does not appear in the list, and the Projector module is not available (in the top orange navigation bar).

However, if I run Tensorboard manually (`tensorboard --logdir venv/.guild/runs/7ac129a6c51d414884179fbe03bd6ea7`), it works fine and the Projector appears.
This is what makes me think it may be a guild issue.

Here is how my Tensorboard `runs` folder structure look like:
```
runs
└── Dec19_13-47-32_ip-172-20-211-222
    ├── 00000
    │   ├── offers_resumes_val
    │   │   ├── metadata.tsv
    │   │   └── tensors.tsv
    │   └── offers_val
    │       ├── metadata.tsv
    │       └── tensors.tsv
    ├── events.out.tfevents.1576763252.ip-172-20-211-222.7638.0
    └── projector_config.pbtxt
```

**Comment by gar1t** [comment]
Thank you for the report! Do you have a simple example that generates the logs above (or similar)? The docs on the projector are thin-to-nonexistent from what I can see.

Offhand it looks like the projection visualization requires `projector_config.pbtxt` - and indeed Guild does not copy that over! The general design philosophy for TensorBoard has historically been to avoid dependencies on files outside the TF event logs so Guild makes that assumption. We'll get this fixed but I'd like to make sure we're getting the right solution in place.

**Comment by tbelhalfaoui** [comment]
@gar1t Thanks for the reply!

Here is a minimal code:
```python
import torch
from torch.utils.tensorboard import SummaryWriter

tensorboard_writer = SummaryWriter()
tensorboard_writer.add_embedding(torch.randn((10, 5)),
                                 metadata=['dummy'] * 10,
                                 tag='embeddings')
```

Indeed, `projector_config.pbtxt` does not get copied over to the `/tmp/guild-tensorboard-8nzdxbvd` directory, which seems to be causing the issue.

**Comment by gar1t** [comment]
Okay, long time coming but this will be fixed in 0.7 - sorry about the eternal delay!

The expectation moving forward is that every plugin works when TensorBoard is run with Guild.

**Comment by gar1t** [comment]
The fix is now generally available in 0.7.0. If this is still an issue, please re-open and we'll sort it out. Thanks again for the report.
